client/ayon_core/tools/tray/lib.py

```diff
@@ -356,6 +356,38 @@ def is_tray_running(
     return state != TrayState.NOT_RUNNING
 
 
+def show_message_in_tray(
+    title, message, icon=None, msecs=None, tray_url=None
+):
+    """Show message in tray.
+
+    Args:
+        title (str): Message title.
+        message (str): Message content.
+        icon (Optional[str]): Icon for the message.
```

Comment:
can we specify the available names in the docstring here?
And is there a type hint possibility where it forces a set of specific strings?

Reply:
>  And is there a type hint possibility where it forces a set of specific strings?

There is Literal (not available in all pythons). But can be used in docstring.